### PR TITLE
Add maxwidth for interactor table columns

### PR DESCRIPTION
**Pull request by jnicklas**
See #552

### CODE REVIEW

**Comment by jnicklas** [comment]
Why not just use an already created ascii table library? I've looked at a few of them, and they just don't seem particularly good. All of them are overcomplicated, and I haven't seen a single one which allows you to set a maximum width, the one feature we'd actually like to have.

File: .changeset/lemon-roses-type.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@bigtest/interactor": patch
+"bigtest": patch
+---
+
+Add maximul column with for interactor alternative suggestion tables
```
Comment (cowboyd) [inline .changeset/lemon-roses-type.md:6]:
```suggestion
Add maximum column with for interactor alternative suggestion tables
```

**Review by cowboyd — COMMENTED** [review]
How do we handle overflow if a value is longer than the column width?

**Comment by jnicklas** [comment]
The value is just truncated with `…` if it overflows the column width.\

Screenshot 2020-10-22 at 17 33 08

**Review by cowboyd — APPROVED** [review]
w00t!

File: .changeset/lemon-roses-type.md (hunk 1 above)
Comment (cowboyd) [reply .changeset/lemon-roses-type.md:6]: @jnicklas is this what you meant? ☝️

**Pull request merged**